lib/galaxy/config.py

```diff
@@ -383,6 +390,14 @@ def __init__( self, **kwargs ):
         self.galaxy_infrastructure_url = galaxy_infrastructure_url
         self.galaxy_infrastructure_url_set = galaxy_infrastructure_url_set
 
+        # Verify host:port is available before continuing
+        try:
+            q = bind(*re.sub(r'http(s?)\://', '', self.galaxy_infrastructure_url).split(':'))
+            q.close()
+        except Exception, e:
```

Comment:
I could throw the specific errno98 socket error or is there a more specific internal exception relating to used resources perhaps?